pkg/istio/rbac/v2/provider_test.go

```diff
@@ -218,10 +220,18 @@ func getExpectedAuthzPolicy() []model.Config {
 							},
 						},
 					},
+					{
+						Source: &v1beta1.Source{
+							Namespaces: []string{
+								"test--domain-namespace2",
+							},
+						},
+					},
 					{
 						Source: &v1beta1.Source{
 							RequestPrincipals: []string{
 								"*",
+								"athenz/test-domain.namespace2.*",
```

Comment:
role for the namespace should not be added as `RequestPrincipals`, can you update code for generating `RequestPrincipals` to exclude namespace role?

Reply:
https://github.com/yahoo/k8s-athenz-istio-auth/pull/58/commits/c2131ccc31aa4edd6d200d83c5067f928a93cb2c